### PR TITLE
adding service

### PR DESCRIPTION
**Pull request by saurabhmishraepam**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-rest</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+		</dependency>
```
Comment (ashoksuryap) [inline pom.xml]: Need this dependency ?

File: mvnw.cmd [hunk 2]
```diff
@@ -0,0 +1,161 @@
+@REM ----------------------------------------------------------------------------
```
Comment (ashoksuryap) [inline mvnw.cmd]: please remove unnecessary files

File: src/main/resources/bootstrap.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: eureka
+  cloud:
+    config:
+      uri: ${CONFIG_SERVER_URL:http://localhost:8888}
```
Comment (ashoksuryap) [inline src/main/resources/bootstrap.yml:6]: Are you using spring cloud config?

File: pom.xml (hunk 1 above)
Comment (saurabhmishraepam) [reply pom.xml]: removed it

File: mvnw.cmd (hunk 2 above)
Comment (saurabhmishraepam) [reply mvnw.cmd]: modified

File: src/main/resources/bootstrap.yml (hunk 3 above)
Comment (saurabhmishraepam) [reply src/main/resources/bootstrap.yml:6]: yes I was using it

**Review by ashoksuryap — APPROVED** [review]

**Pull request merged**